docs/newbs_getting_started.md

```diff
@@ -164,7 +164,7 @@ In most situations you will want to answer `y` to all of the prompts.
 
 ## 4. Test Your Build Environment
 
-Now that your QMK build environment is set up, you can build a firmware for your keyboard. Start by trying to build the keyboard's default keymap. You should be able to do that with a command in this format:
+Now that your QMK build environment is set up, you can build a firmware for your keyboard. Start by trying to build the keyboard's default keymap. If you're not sure what your keyboard's name is, check the [keyboards folder](https://github.com/qmk/qmk_firmware/tree/master/keyboards) in the QMK source code. You should be able to build a firmware with a command in this format:
```

Comment:
I'd say that this would go better at the end of the block (before the next section).  And more markdown friendly. Eg:

```markdown
?> If you're not sure what your keyboard's name is, you can check the [keyboards folder](https://github.com/qmk/qmk_firmware/tree/master/keyboards) in the QMK Firmware repo. 


Also, if you add `util/qmk_tab_completion.sh` to your profile/rc file, you can enable tab completion for the qmk commands.